examples/warm_up_example/fedavg_warm_up/client.py

```diff
@@ -30,9 +30,11 @@ def __init__(
         metrics: Sequence[Metric],
         device: torch.device,
         checkpoint_dir: str,
+        client_name: str,
```

Comment:
Making this an argument you pass is easier to manage than leaving it up to the generate hash and trying to match things.